apps/judicial-system/web/src/utils/formHelper.ts

```diff
@@ -50,10 +50,15 @@ export const validateAndSetErrorMessage = (
 ) => {
   const validation = validations.validate([[value, validationsToRun]])
 
-  if (!validation.isValid && errorMessageSetter) {
+  if (errorMessageSetter) {
     errorMessageSetter(validation.errorMessage)
-    return
   }
+
+  if (!validation.isValid) {
+    return false
+  }
+
+  return true
```

Comment:
Consider simplifying:

```suggestion
  return validation.isValid
```